plugin/src/main/java/com/denizenscript/denizen/events/entity/ExpBottleBreaksScriptEvent.java

```diff
@@ -0,0 +1,81 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.containers.ScriptContainer;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.ExpBottleEvent;
+
+public class ExpBottleBreaksScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // exp bottle breaks
+    //
+    // @Regex ^on exp bottle breaks$
+    // @Switch in <area>
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a thrown exp bottle breaks.
+    //
+    // @Context
+    // <context.bottle> returns the EntityTag of the thrown exp bottle.
+    // <context.experience> returns the amount of experience to be spawned.
+    // <context.show_effect> returns whether the effect should be shown.
+    //
+    // -->
+
+    public ExpBottleBreaksScriptEvent() {
+        instance = this;
+    }
+
+    public static ExpBottleBreaksScriptEvent instance;
+    private EntityTag bottle;
+    private ElementTag experience;
+    private ElementTag showEffect;
+    private ExpBottleEvent event;
+
+    @Override
+    public boolean couldMatch(ScriptContainer scriptContainer, String s) {
+        String lower = CoreUtilities.toLowerCase(s);
+        return (lower.equals("exp bottle breaks"));

```

Comment:
Please use the modern `couldMatch(ScriptPath path)` setup that most events now use.